source/utilities.cc

```diff
@@ -1518,17 +1545,24 @@ namespace aspect
            * Constructor, see resize()
            */
           band_matrix(int dim, int n_u, int n_l);
+          
+          
+          
           /**
            * Resize to a @p dim by @dim matrix with given number
            * of off-diagonals.
            */
           void resize(int dim, int n_u, int n_l);
 
+
+
           /**
            * Return the dimension of the matrix
            */
           int dim() const;
```

Comment:
Ah, you couldn't know -- we only apply the three-line rule to global functions but not member functions :-)